### DISCUSSION TITLE
%E6%A8%A1%E5%9E%8B%E9%83%A8%E7%BD%B2/20230114%E5%8D%95%E6%9C%BA%E5%A4%9A%E5%8D%A1%E8%AE%A1%E7%AE%97%E8%B5%84%E6%BA%90%E5%88%86%E9%85%8D/

### DISCUSSION THREAD

**Discussion by giscus** [thb1314/thb1314.github.io#5] [Announcements]
# %E6%A8%A1%E5%9E%8B%E9%83%A8%E7%BD%B2/20230114%E5%8D%95%E6%9C%BA%E5%A4%9A%E5%8D%A1%E8%AE%A1%E7%AE%97%E8%B5%84%E6%BA%90%E5%88%86%E9%85%8D/

https://www.hbblog.cn/%E6%A8%A1%E5%9E%8B%E9%83%A8%E7%BD%B2/20230114%E5%8D%95%E6%9C%BA%E5%A4%9A%E5%8D%A1%E8%AE%A1%E7%AE%97%E8%B5%84%E6%BA%90%E5%88%86%E9%85%8D/

**Comment by Twenty3hree** [comment]
博主写的很好，只是最后是不是2023年呐，晃眼看还以为是十年前的文章，细想又有些不对

**Reply by thb1314** [reply]
收到，感谢指正，下次更新文章我就修改一下。
